### PR TITLE
update client

### PR DESCRIPTION
**Pull request by cristianvasquez**
Update the SPARQL http client

### CODE REVIEW

**Comment by cristianvasquez** [comment]
@ludovicm67 this is a clean one

File: index.js [hunk 1]
```diff
@@ -111,7 +104,7 @@ export class SparqlHandler {
   async head (_req, res, next, iri) {
     iri = encodeURI(iri)
 
-    debug('handle OPTIONS request for IRI <' + iri + '>')
+    debug('handle HEAD request for IRI <' + iri + '>')
```
Comment (ludovicm67) [inline index.js:107]: Nice catch! 👍

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "debug": "^4.3.4",
     "node-fetch": "^3.3.0",
```
Comment (ludovicm67) [inline package.json]: The module `node-fetch` is not required anymore as I can see, no?

```suggestion
```

**Comment by cristianvasquez** [comment]
It's not needed, you're right

**Comment by cristianvasquez** [comment]
removed.

**Review by ludovicm67 — APPROVED** [review]
Thanks! Looks great 👍

**Pull request merged**